tests/test_base_device.py

```diff
@@ -1081,6 +1081,9 @@ def test_estimator_with_different_pauli_obs(self, mocker, wire, angle, op, expec
         ],
     )
     @flaky(max_runs=10, min_passes=7)
+    @pytest.mark.xfail(
+        reason="Qiskit variances is different from PennyLane variances, discussion pending on resolution"
+    )
```

Comment:
We xfail these tests because the qiskit's std is different from what is expected.

Long story short: if you compute var(O) manually using pennylane/expectation values using `<O^2> - <O>^2` where O is the operator, and then calculate the `std` using sqrt(var(O))/sqrt(N), you will get a different `std` from Qiskit 1.2.0. In Qiskit <1.1.0, you will get the same `std` for commuting observables but not for non-commuting observables.

If O is a linear combination of pauli terms (Pi, Pj...), Qiskit computes the variance as Var(Pi) + Var(Pj) + ..., which doesn't seem right to me. Perhaps they're doing some kind of approximation?

Reply:
Sounds good to me.